.gitmodules

```diff
@@ -1,6 +1,6 @@
-[submodule "lib/forge-std"]
-	path = lib/forge-std
+[submodule "forge/forge-std"]
+	path = forge/forge-std
```

Comment:
This is not correct, as this local path doesn't exist. The correct one should be `lib/forge-std`. Build on our PR is failing because of not being able to checkout submodules due to this issue. (https://github.com/0xPolygon/polygon-edge/actions/runs/4753471533/jobs/8445099667)